.coveragerc

```diff
@@ -0,0 +1,4 @@
+[report]
+
+exclude_also =
+    if tp.TYPE_CHECKING:
```

Comment:
I noticed you added `# pragma: no cover` on every line inside type-checking blocks. This config file specification means you don't have to do that!

Reply:
Awesome! I wish I had this earlier...